src/coreclr/jit/compphases.h

```diff
@@ -45,6 +45,7 @@ CompPhaseNameMacro(PHASE_EARLY_UPDATE_FLOW_GRAPH,    "Update flow graph early pa
 CompPhaseNameMacro(PHASE_DFS_BLOCKS,                 "DFS blocks and remove dead code",false, -1, false)
 CompPhaseNameMacro(PHASE_DFS_BLOCKS2,                "DFS blocks and remove dead code 2",false, -1, false)
 CompPhaseNameMacro(PHASE_STR_ADRLCL,                 "Morph - Structs/AddrExp",        false, -1, false)
+CompPhaseNameMacro(PHASE_LCL_MASKS,                  "Local masks",                    false, -1, false)
```

Comment:
I think the name is too generic. Can we make it something about optimizing vector<->mask conversions?

Reply:
How about:

`PHASE_OPTIMIZE_MASK_CONVERSIONS`
`Compiler::fgOptimizeMaskConversions()`
`optimizeMaskConversions.cpp`

?

Reply:
That looks good to me (with "optimizemaskconversions.cpp" as the name, as file names in the JIT are all lower case)